fuzzers/libfuzzer/fuzzer.py

```diff
@@ -56,8 +62,10 @@ def fuzz(input_corpus, output_corpus, target_binary):
         # using it will cause libFuzzer to find "crashes" no one cares about.
         '-detect_leaks=0',
     ]
+    flags += extra_flags
     if 'ADDITIONAL_ARGS' in os.environ:
         flags += os.environ['ADDITIONAL_ARGS'].split(' ')
 
+    print('[run_fuzzer] Running target with libFuzzer')
     command = [target_binary, output_corpus, input_corpus] + flags
     subprocess.call(command)
```

Comment:
Is this CL rebased, i remember there was a print i added for printing the command, lets not overwrite that.